python/ray/tests/test_object_spilling.py

```diff
@@ -16,15 +16,18 @@ def test_spill_objects_manually(shutdown_only):
     # Limit our object store to 75 MiB of memory.
     ray.init(
         object_store_memory=75 * 1024 * 1024,
-        _object_spilling_config={
-            "type": "filesystem",
-            "params": {
-                "directory_path": "/tmp"
-            }
-        },
         _system_config={
             "object_store_full_max_retries": 0,
+            "automatic_object_spilling_enabled": False,
             "max_io_workers": 4,
+            "object_spilling_config": json.dumps(
+                {
+                    "type": "filesystem",
+                    "params": {
+                        "directory_path": "/tmp"
```

Comment:
Can we use https://docs.pytest.org/en/stable/tmpdir.html to clean this up?

Reply:
Er I'm not sure how to do that. We need to pass a JSON object here. Is there a way to get the pathname of the tmp dir?

Reply:
Yeah, I think you can get the absolute path for use in a test (or delete a tempdir in a try finally block).